10-JavaScript/welcome-to-js.md

```diff
@@ -214,7 +229,7 @@ Looks similar to a conditional! This syntax comes from the programming language
 Here's a realistic example:
```

Comment:
I think this would also be a good opportunity to highlight the other good types of `for` loops